### DISCUSSION TITLE
Version 1.1 Release Candidate 1

### DISCUSSION THREAD

**Discussion by ErinaSugino** [ErinaSugino/Starbound-SBR-Mod#56] [Update Builds]
## Version 1.1 Patchnotes
- Integrated Lustbound into SBR
- Added modular pregnancy management (Including multi-baby pregnancies)
- Added new cumflation and fertility stacking mechanics (Thanks to XspeedPL)
- Added properly working baby genetics
- Re-added global NPC auto-climax
- Outsourced UI sync data for easy patching
- Added stillborn chance and incest birthrate penalty
- Added True Kid NPCs
- Fixed incompatibility with "Npc Drops Gear"
- Smarter, configurable, non-perma-stuck NPC arousal behavior
- Kid and family status abiding conversation behavior

## RC1 Notes
- Exhaustive testing needed before release.

### Important Notes
- Do not use on main savefile - 1.1 changes the data format for pregnancies which is not backwards compatible. Reverting back to 1.0 will erase or break all ongoing pregnancies!

<hr /><em>This discussion was created from the release <a href='https://github.com/ErinaSugino/Starbound-SBR-Mod/releases/tag/v1.1_rc1'>Version 1.1 Release Candidate 1</a>.</em>

**Comment by Mixarar** [comment]


**Reply by ErinaSugino** [reply]
Not gonna rollout a new RC just for this, so just manually apply the change from this commit and see if it fixes it. 7234b8b

As for statistics - are you using the item or something like QuickbarMini/Stardust Core Lite?

**Reply by ErinaSugino** [reply]
That's known, and there is no way to make it work without the devs of that mod changing their inner workings.

**Reply by ErinaSugino** [reply]
Actually, wait. 061d13f

**Comment by ghost** [comment]


**Reply by ErinaSugino** [reply]
Even though this is - or should I say *especially because this is* - the discussion thread for the Release Candidate this does not mean that basic bug reporting procedures are not needed.

That means, hand over the log file. And make sure you have turned on info and debug level logging. Otherwise all I can say to your description is "Works for me ¯\\\_(ツ)\_/¯".

**Reply by ghost** [reply]
I found my issue but thank you for the work you and others put into these mods.
